core/src/main/scala/org/apache/spark/scheduler/Stage.scala

```diff
@@ -145,6 +146,6 @@ private[scheduler] abstract class Stage(
 }
 
 private[scheduler] object Stage {
-  // The number of consecutive failures allowed before a stage is aborted
-  val MAX_CONSECUTIVE_FETCH_FAILURES = 4
+  // The number of consecutive fetch failures allowed before a stage is aborted
+  val DEFAULT_MAX_CONSECUTIVE_FETCH_FAILURES = 4
```

Comment:
Given that this is used in the DAGScheduler, not here, I think it makes more sense to put it in the DAGScheduler object.